docs/dev/table/connect.md

```diff
@@ -1075,6 +1075,72 @@ CREATE TABLE MyUserTable (
 
 {% top %}
 
+### HBase Connector
+
+<span class="label label-primary">Source: Batch</span>
+<span class="label label-primary">Sink: Batch</span>
+<span class="label label-primary">Sink: Streaming Append Mode</span>
+<span class="label label-primary">Sink: Streaming Upsert Mode</span>
+<span class="label label-primary">Temporal Join: Sync Mode</span>
+
+The HBase connector allows for reading from an HBase cluster.
+The HBase connector allows for writing into an HBase cluster.
+
+The connector can operate in [upsert mode](#update-modes) for exchanging UPSERT/DELETE messages with the external system using a [key defined by the query](./streaming/dynamic_tables.html#table-to-stream-conversion).
```

Comment:
```suggestion
The connector can operate in [upsert mode](#update-modes) by executing UPSERT and DELETE statements on the cluster using a [key defined by the query](./streaming/dynamic_tables.html#table-to-stream-conversion).
```

Reply:
This is same to Elasticsearch, it is messages instead of statements.